tests/moduletool/test_install.py

```diff
@@ -255,6 +255,44 @@ def test_module_install_version(
     assert mod.version == module_version
 
 
+@pytest.mark.slowtest
+def test_module_install_reinstall(
+    tmpdir: py.path.local,
+    snippetcompiler_clean,
+    modules_v2_dir,
+) -> None:
+    """
+    Verify that reinstalling a module from source without bumping the version installs any changes to model and Python files.
+    """
+    module_name: str = "minimalv2module"
+    module_path: str = str(tmpdir.join(module_name))
+    shutil.copytree(os.path.join(modules_v2_dir, module_name), module_path)
+
+    # set up simple project and activate snippetcompiler venv
+    project: module.Project = snippetcompiler_clean.setup_for_snippet("")
+    os.chdir(project.path)
```

Comment:
This call is not required. The snippetcompiler does it.

Reply:
Thanks, I've removed it.